arbnode/batch_poster.go

```diff
@@ -1153,21 +1155,16 @@ func (b *BatchPoster) Start(ctxIn context.Context) {
 		}
 		posted, err := b.maybePostSequencerBatch(ctx)
 		if err == nil {
-			b.firstEphemeralError = time.Time{}
+			commonEphemeralErrorHandler.Reset()
+			exceedMaxMempoolSizeEphemeralErrorHandler.Reset()
 		}
 		if err != nil {
 			b.building = nil
 			logLevel := log.Error
 			// Likely the inbox tracker just isn't caught up.
 			// Let's see if this error disappears naturally.
-			if b.firstEphemeralError == (time.Time{}) {
-				b.firstEphemeralError = time.Now()
-				logLevel = log.Warn
-			} else if time.Since(b.firstEphemeralError) < time.Minute {
-				logLevel = log.Warn
-			} else if time.Since(b.firstEphemeralError) < time.Minute*5 && strings.Contains(err.Error(), "will exceed max mempool size") {
-				logLevel = log.Warn
-			}
+			logLevel = commonEphemeralErrorHandler.LogLevel(err, logLevel)
+			logLevel = commonEphemeralErrorHandler.LogLevel(err, logLevel)
```

Comment:
I think there's a copy-paste mistake here, the second one looks like it should be `exceedMaxMempoolSizeEphemeralErrorHandler`

Reply:
fixed it